<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpellReview</class>
 <widget class="QDialog" name="SpellReview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(81, 81, 81);</string>
  </property>
  <widget class="QLabel" name="labelReview">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>276</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>是否进入复习模式？</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWrongNum">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>错误：20</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRightNum">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>正确：12</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>测试结束！</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonYes">
   <property name="geometry">
    <rect>
     <x>191</x>
     <y>290</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>是</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTipNum">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>提示：18</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNo">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>290</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>否</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTotalNum">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>总计：50</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
